kdumpctl

```diff
@@ -1729,8 +1758,27 @@ main()
 			reset_crashkernel_for_installed_kernel "$2"
 		fi
 		;;
+	test)
+		is_kernel_loaded "$DEFAULT_DUMP_MODE"
+		if [[ "$?" -ne 0 ]]; then
+			derror "Kdump should be operational before test."
+			exit 1
+		fi
+		read -p "DANGER!!! Will perform a kdump test by crashing the system, proceed? (Y/N): " input
```

Comment:
Two things:
1) I prefer giving the different options as lower case letters and indicate the default as upper case letter. I.e. 's;Y/N;y/N;'
2) I understand that you want to give users a chance to abort in case they have second thoughts. But that only makes sense when users trigger the test manually. When they use it in a test script this will cause issues. Does it make sense to add a --force option to disable this question?

Reply:
Yes, I agree with your "--force" option, it is better!